src/searchParameters/searchParameters.js

```diff
@@ -6433,7 +6431,7 @@ const searchParameterQueries = {
 		'name': {
 			'description': 'A server defined search that may match any of the string fields in the HumanName, including family, give, prefix, suffix, suffix, and/or text',
 			'type': 'string',
-			'field': 'name',
+			'fields': ['name.text', 'name.given', 'name.family', 'name.suffix', 'name.prefix'],
```

Comment:
Could we make this a variable that we can reuse and change in one place vs many?